.github/workflows/ci_tests.yaml

```
@@ -105,7 +105,8 @@ jobs:
         os: [ubuntu-latest]
     env:
       # LD_LIBRARY_PATH: ${{ github.workspace }}/gmt/lib:$LD_LIBRARY_PATH
-      GMT_LIBRARY_PATH: ${{ github.workspace }}/gmt/lib
+      GMT_INSTALL_DIR: ${{ github.workspace }}
+      GMT_LIBRARY_PATH: ${{ github.workspace }}/gmt-install-dir/lib
```

I think you should change it to this:
```suggestion
      GMT_INSTALL_DIR: ${{ github.workspace }}/gmt-install-dir
      GMT_LIBRARY_PATH: ${{ github.workspace }}/gmt-install-dir/lib
```